### ISSUE TITLE
Will not show correct class...

### ISSUE THREAD

**Issue by CountVooku** [Nyyrazzilyss/NyyLIB#38]
Was doing it on 7/31/17 version and still doing it on 8/28/17 version. "Who" doesn't correct...

![classissue](https://user-images.githubusercontent.com/28542037/29892913-53f88fa0-8d9e-11e7-97e8-df4ecc0febfb.JPG)

**Comment by Nyyrazzilyss** [comment]
I had fixed this 3 months ago for when the mud was reporting a different name/class combination then the database, it changed the database to the new information.

The trigger that checks this though is on the 'who' command, it doesn't check on 'who charname'. I will add that. Additionally, I doubt it corrects race just class.

However, like I said I thought I fixed this 3 months ago. This is the first re-occurance i've seen of this particular problem, numerous people had it in the past and no-one else since i'd changed it. Are you sure you're using the package in a profile that doesn't have any pre-existing changes of your own in it?
